src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx

```diff
@@ -344,7 +344,7 @@ const TransferFundsDialogForm = ({
           />
         </div>
       </DialogSection>
-      <DialogSection appearance={{ align: 'right' }}>
+      <DialogSection appearance={{ align: 'right', theme: 'footer' }}>
```

Comment:
The confirm button needs to be wider. Currently yours is `120px`, and it needs to be `160px`.

Look up how we implemented this in `OrganizationAddDomains`: 
https://github.com/JoinColony/colonyDapp/blob/05131f8ce4c9e23793d2ada77ba7026a318f4ff2/src/modules/admin/components/Domains/OrganizationAddDomains.tsx#L70

By overwriting the component styles directly, with a value set in the css modules, using `@value`:
https://github.com/JoinColony/colonyDapp/blob/05131f8ce4c9e23793d2ada77ba7026a318f4ff2/src/modules/admin/components/Domains/OrganizationAddDomains.css#L3

Reply:
Mmmm I think it's gonna be better if we update the size of large buttons. According to the style guide, the large button is 160x44 so by updating the padding applied by the appearance, we should be fine

Reply:
Not necesarily. 

The `large` button theme is in use elsewhere as well. So if you change that, you'll modify a lot of places in the app we might not want to change.

That's why I suggested you make the change locally just for the dialog.